lib/topological_inventory.rb

```diff
@@ -16,6 +16,7 @@ def self.call
     TopologicalInventoryApiClient.configure do |config|
       config.host = ENV['TOPOLOGICAL_INVENTORY_URL'] || 'localhost'
       config.scheme = URI.parse(ENV['TOPOLOGICAL_INVENTORY_URL']).try(:scheme) || 'http'
+      config.base_path = ENV['TOPOLOGICAL_INVENTORY_BASE_PATH'] || '/api/topological-inventory/v0.1'
```

Comment:
I was thinking of using `PATH_PREFIX` here. It's fair to assume that if you're changing the path prefix then the other apps are too, right?

So something like:
```ruby
config.base_path = ENV['PATH_PREFIX'].present? ? File.join("/", ENV["PATH_PREFIX"], "topological-inventory", "v0.1") : "/api/topological-inventory/v0.1"
```